spec/features/pagination_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.feature "Pagination", type: :feature do
+  let(:organisation) { create(:organisation_with_content_items, content_items_count: 3) }
+
+  before do
+    Kaminari.configure do |config|
+      @default_per_page = config.default_per_page
+      config.default_per_page = 2
+    end
+  end
+
+  after do
+    Kaminari.configure do |config|
+      config.default_per_page = @default_per_page
+    end
+  end
+
+  scenario "When the user goes to the first page of content items for an organisation, the current page number is 1" do
+    visit "organisations/#{organisation.id}/content_items"
+    expect(page).to have_selector('.pagination .current', text: 1)
+  end
+
+  scenario "When user navigates to the last page of the list, the current page number is 2" do
+    visit "organisations/#{organisation.id}/content_items"
+    within ("nav.pagination") do
+      click_on "Last"
+    end
+
+    expect(page).to have_selector('.pagination .current', text: 2)
+  end
+
+  scenario "When user navigates to the next page, the current page number is 2" do
+    visit "organisations/#{organisation.id}/content_items"
+    click_on "Next"
+
+    expect(page).to have_selector('.pagination .current', text: 2)
+  end
+
+  scenario "When the user can navigate to the previous page from page 2, the current page number is 1" do
+    visit "organisations/#{organisation.id}/content_items?page=2"
+    click_on "Prev"
+
+    expect(page).to have_selector('.pagination .current', text: 1)
+  end
+
+  scenario "When the user goes to any page of content items for an organisation, they can see the total number of content items" do
+    visit "organisations/#{organisation.id}/content_items"
+    expect(page).to have_selector('div#total-content-items', text: 'Displaying content items 1 - 2 of 3 in total')
+  end
+end
```

Comment:
I would try to improve a bit the feature test description. 

```
± % be rspec --format documentation spec/features                                                                                                                                                                                      !10247

Randomized with seed 42722

Pagination
  When the user goes to the first page of content items for an organisation, the current page number is 1
  When user navigates to the next page, the current page number is 2
  When user navigates to the last page of the list, the current page number is 2
  When the user can navigate to the previous page from page 2, the current page number is 1
  When the user goes to any page of content items for an organisation, they can see the total number of content items

Finished in 4.89 seconds (files took 4.54 seconds to load)
5 examples, 0 failures

Randomized with seed 42722
```

What do you think of this description?

```
Randomized with seed 37453

Content Items Pagination
  The user can see a message with the total number of content items
  When user navigates to
    the first page, the current page number is 1
    the previous page from page 2, the current page number is 1
    the last page, the current page number is 2
    the next page, the current page number is 2

Finished in 1.42 seconds (files took 4.19 seconds to load)
5 examples, 0 failures

Randomized with seed 37453
```

This would be the ruby code for it:

```ruby
require 'rails_helper'

RSpec.feature 'Content Items Pagination', type: :feature do
  let(:organisation) { create(:organisation_with_content_items, content_items_count: 3) }

  before do
    Kaminari.configure do |config|
      @default_per_page       = config.default_per_page
      config.default_per_page = 2
    end
  end

  after do
    Kaminari.configure do |config|
      config.default_per_page = @default_per_page
    end
  end

  context 'When user navigates to' do

    scenario 'the first page, the current page number is 1' do
      visit "organisations/#{organisation.id}/content_items"

      expect(page).to have_selector('.pagination .current', text: 1)
    end

    scenario 'the last page, the current page number is 2' do
      visit "organisations/#{organisation.id}/content_items"
      within ('nav.pagination') do
        click_on 'Last'
      end

      expect(page).to have_selector('.pagination .current', text: 2)
    end

    scenario 'the next page, the current page number is 2' do
      visit "organisations/#{organisation.id}/content_items"
      click_on 'Next'

      expect(page).to have_selector('.pagination .current', text: 2)
    end

    scenario 'the previous page from page 2, the current page number is 1' do
      visit "organisations/#{organisation.id}/content_items?page=2"
      click_on 'Prev'

      expect(page).to have_selector('.pagination .current', text: 1)
    end
  end


  scenario 'The user can see a message with the total number of content items' do
    visit "organisations/#{organisation.id}/content_items"

    expect(page).to have_selector('div#total-content-items', text: 'Displaying content items 1 - 2 of 3 in total')
  end
end
```